src/libraries/pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -51,12 +46,17 @@
     <NS21PrereleaseLibraryPackage Include="System.Reflection.Context" />
 
     <!-- Packages we don't build in main anymore -->
+    <LibraryPackage Include="Microsoft.Win32.Registry" Version="5.0.0" />
     <LibraryPackage Include="System.Data.DataSetExtensions" Version="4.5.0" />
     <LibraryPackage Include="System.Data.SqlClient" Version="4.8.2" />
+    <LibraryPackage Include="System.IO.FileSystem.AccessControl" Version="5.0.0" />
+    <LibraryPackage Include="System.IO.Pipes.AccessControl" Version="5.0.0" />
     <LibraryPackage Include="System.Reflection.Emit" Version="4.7.0" />
     <LibraryPackage Include="System.Reflection.Emit.ILGeneration" Version="4.7.0" />
     <LibraryPackage Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
+    <LibraryPackage Include="System.Security.AccessControl" Version="5.0.0" />
     <LibraryPackage Include="System.Security.Cryptography.Cng" Version="5.0.0" />
+    <LibraryPackage Include="System.Security.Principal.Windows" Version="5.0.0" />
```

Comment:
maybe we should add a 6.0 dependency group that doesn't list these packages?

Reply:
Thanks that was a great suggestion.